### PR TITLE
[Rename-registry-like-class-DDM]Lps 167105

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-167105

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java [hunk 1]
```diff
@@ -265,12 +265,12 @@ private JSONObject _processDDMFormValues(
...
-	private final DDMFormFieldTypeReportProcessorTracker
-		_ddmFormFieldTypeReportProcessorTracker =
-			new DDMFormFieldTypeReportProcessorTracker();
+	private final DDMFormFieldTypeReportProcessorRegistry
+			_ddmFormFieldTypeReportProcessorRegistry =
+			new DDMFormFieldTypeReportProcessorRegistry();
 	private final JSONFactory _jsonFactory;
 
-	private class DDMFormFieldTypeReportProcessorTracker {
+	private class DDMFormFieldTypeReportProcessorRegistry {
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java:273]: @slnn Looking at the logic, this inner class is actually a "factory", not a "registry". Also, it's only used by itself once, so I don't think a inner class is needed. Just move the method up, make it private, and call it.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java (hunk 1 above)
Comment (slnn) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java]: Done and has forced push to this pull

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@slnn semver error:

```
  [exec] com.liferay.semantic.versioning.SemanticVersioningTest > testSemanticVersioning[/apps/data-engine/data-engine-rest-api] FAILED
     [exec]     java.lang.AssertionError:   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec]     = ================================================== ========== ========== ========== ========== ==========
     [exec]     * com.liferay.data.engine.rest.dto.v2_0.util         MAJOR      1.0.0      1.0.0      2.0.0      VERSION INCREASE REQUIRED
```
full log [here](https://testray.liferay.com/reports/production/logs/2022-11/test-1-39/test-portal-acceptance-pullrequest(master)/1583/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

**Comment by dantewang** [comment]
@slnn Could you double check the sem ver error and resend? Thanks!

**Comment by slnn** [comment]
Update the semantic version and resend in https://github.com/liferay-core-infra/liferay-portal/pull/2433

**Pull request closed without merging**